.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: node_js
+
 node_js:
-  - "0.10"
-  - "0.8"
+  - "0.12"
+  - "iojs"
```

Comment:
Yep, looks like the first version will show up in September. I'll update the `.travis.yml` as soon as a version is available.